sequence/src/main/java/com/vitreoussoftware/bioinformatics/sequence/encoding/IupacEncodingScheme.java

```diff
@@ -5,29 +5,48 @@
 
 
 /**
```

Comment:
I had IupacEncodingScheme implement its complement function in terms of the ExpandedIupacEncodingScheme as an implementation detail. However re-use here gets tricky. ExpandedIupacEncodingScheme cannot be used "as-a" IupacEncodingScheme (or vice-versa) because of its extra possible value. 

I'll give it some more thought though.